doc/usage/configuration.rst

```diff
@@ -1447,8 +1448,8 @@ Options for warning control
       Added ``misc.copy_overwrite``.
 
    .. versionadded:: 8.2
-      Added ``duplicate_declaration.c`` and ``duplicate_declaration.cpp``.
-
+      Added ``duplicate_declaration.c`` and ``duplicate_declaration.cpp``, and
+      ``toc.glob_not_matching``.
```

Comment:
Please split this into a new `versionadded` directive

Reply:
Done.